crowbar_framework/lib/crowbar/product.rb

```diff
@@ -29,7 +29,7 @@ def self.is_ses?
 
     def self.ses_platform
       # Returns the platform used for SES
-      "suse-12.2"
+      "suse-12.3"
```

Comment:
no, its sp2, it means mkcloud needs to deploy ceph nodes with sp2 then (I think we used to have code for that)